### PR TITLE
231 move souts to controller

### PR DESCRIPTION
**Pull request by VladaKmets**

### CODE REVIEW

File: src/main/java/com/vlada/tasks/finalgrade/controller/FinalGradeController.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        if (inputString.equals("F")) {
+            BufferedReader bufferedReader = null;
+            FileReader fileReader = null;
+            try {
+                fileReader = new FileReader("finalgrade.txt");
+                bufferedReader = new BufferedReader(fileReader);
+                String sCurrentLine;
+                sCurrentLine = bufferedReader.readLine();
+                String[] subStr;
+                subStr = sCurrentLine.split(Constants.DELIMETER);
```
Comment (MaksymKmets) [inline src/main/java/com/vlada/tasks/finalgrade/controller/FinalGradeController.java]: inline and rename with full sense (subStr->subStrings)

File: src/main/java/com/vlada/tasks/alarm/controller/AlarmController.java [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+                        bufferedReader.close();
+                    }
+                    if (fileReader != null) {
+                        fileReader.close();
+                    }
+                } catch (IOException e) {
+                    System.out.println("Something went wrong. Contact admin.");
+                }
+            }
+        } else if (inputString.equals("I")) {
```
Comment (MaksymKmets) [inline src/main/java/com/vlada/tasks/alarm/controller/AlarmController.java:43]: In the class Constants create constants I and F with the same values. Use this constants during equals operations in each controller

File: src/main/java/com/vlada/tasks/finalgrade/controller/FinalGradeController.java (hunk 1 above)
Comment (MaksymKmets) [reply src/main/java/com/vlada/tasks/finalgrade/controller/FinalGradeController.java]: also check all variables in request and rename with full sense

File: src/main/java/com/vlada/tasks/finalgrade/controller/FinalGradeController.java [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+import com.vlada.tasks.utils.Constants;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.Scanner;
+
+public class FinalGradeController {
+    public void view() {
+        Scanner keyboard = new Scanner(System.in);
```
Comment (MaksymKmets) [inline src/main/java/com/vlada/tasks/finalgrade/controller/FinalGradeController.java:13]: Don't forget to close the Scanner object. In this situation you can do it in finally block

File: src/main/java/com/vlada/tasks/rotation/controller/RotationController.java [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+        Rotation.generateArray(someArray, arraySize);
+        System.out.println(Arrays.deepToString(someArray));
+        System.out.println();
+        System.out.println("Please, enter the number of times to rotate the array");
+        int rotationTimes = Rotation.getIntFromUser(keyboard);
+        Integer[] rotatedToTheRightArray = Rotation.copyArray(someArray);
+        Integer[] rotatedToTheLeftArray = Rotation.copyArray(someArray);
+        System.out.println("Please, specify how to rotate an array: right(R) or left(L)");
+        String rotation = keyboard.next();
+        if (rotation.equals("R")) {
```
Comment (MaksymKmets) [inline src/main/java/com/vlada/tasks/rotation/controller/RotationController.java]: Also add R and L to Constants and reuse

File: src/main/java/com/vlada/tasks/alarm/controller/AlarmController.java (hunk 2 above)
Comment (VladaKmets) [reply src/main/java/com/vlada/tasks/alarm/controller/AlarmController.java:43]: Done

Comment (VladaKmets) [reply src/main/java/com/vlada/tasks/alarm/controller/AlarmController.java]: Also created constants for options "Right" and "Left" for some classes

File: src/main/java/com/vlada/tasks/rotation/controller/RotationController.java (hunk 4 above)
Comment (VladaKmets) [reply src/main/java/com/vlada/tasks/rotation/controller/RotationController.java]: Done

File: src/main/java/com/vlada/tasks/finalgrade/controller/FinalGradeController.java (hunk 1 above)
Comment (VladaKmets) [reply src/main/java/com/vlada/tasks/finalgrade/controller/FinalGradeController.java]: Done

File: src/main/java/com/vlada/tasks/finalgrade/controller/FinalGradeController.java (hunk 3 above)
Comment (VladaKmets) [reply src/main/java/com/vlada/tasks/finalgrade/controller/FinalGradeController.java:13]: Done

**Pull request merged**